tests/integration/baseTestAPI.py

```diff
@@ -167,6 +167,7 @@ def setUpClass(cls):
         cls.fileStatusDict = lookups.FILE_STATUS_DICT
         cls.ruleSeverityDict = lookups.RULE_SEVERITY_DICT
         cls.errorTypeDict = lookups.ERROR_TYPE_DICT
+        cls.publishStatusDict = lookups.PUBLISH_STATUS_DICT
```

Comment:
Curious, why set this as a class variable? Seems simple enough to access `PUBLISH_STATUS_DICT` directly?